docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -72,6 +72,9 @@ The following sections outline the commands that you can run against Grafana Mim
 
 The following commands interact with Grafana Mimir Alertmanager configuration and alert template files.
 
+> **Note:** The Mimir Alertmanager exposes the configuration API according to the path set by the `-server.path-prefix` flag. It does not use the path set by the `-http.alertmanager-http-prefix` flag.
```

Comment:
```suggestion
> **Note:** The Mimir Alertmanager exposes the configuration API based on the path that you set via the `-server.path-prefix` flag; it does not use the path of the `-http.alertmanager-http-prefix` flag.
```
I am not sure if the user has set both of these paths or one of them. Therefore, I changed this to possession (the path of the ...).